### DISCUSSION TITLE
queue the tasks Using Quartz

### DISCUSSION THREAD

**Discussion by sanjaysingharcadix** [quartznet/quartznet#1754] [Q&A]
Hi ,

I need to Schedule a task by queue. Is that any way the implement the queue task using Quartz.NET.
so when I add the task in queue it will automatic schedule that task which will executed by Quartz.NET

can you please provide some link or suggestion ?

**Comment by lahma** [comment]
Wouldn't you be better served with some service bus implementation like [MassTransit](http://masstransit-project.com/) or similar if you don't need scheduling?
